jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -559,26 +565,26 @@ public void run() {
 		}, 1, TimeUnit.SECONDS);
 	}
 
-	@Override
-	public void fileModified(File file) throws Exception {
+	private void handleFileEvent(File file, String event) throws Exception {
 		if (this.isProcessConfigFile(file)) {
-			log.info("Configuration file modified: " + file);
+			log.info("Configuration file " + event + ": " + file);
```

Comment:
Could we you `{}` placeholders instead of concatenation

Reply:
Fixed.